scripts/skills/actives/rf_command_skill.nut

```diff
@@ -75,12 +75,19 @@ this.rf_command_skill <- this.inherit("scripts/skills/skill", {
 	{
 		local target = _targetTile.getEntity();
 		::Tactical.TurnSequenceBar.moveEntityToFront(target.getID());
+		if (!_user.isHiddenToPlayer())
```

Comment:
Shouldn't this require both the user and the target to be visible? That seems consistent with other log entries e.g. for vanilla attacks.

Reply:
goblin night vision spell only requires the user to be visible
![image](https://github.com/Battle-Modders/mod-reforged/assets/2252464/6ff9ba70-86ad-4986-b877-29f8d37ec3ac)

possess does not care who is visible
![image](https://github.com/Battle-Modders/mod-reforged/assets/2252464/90cc5f60-de36-4f27-a2ba-0fed52050312)

I prefer when the player has to fight a bit more for such information on a battle. And that means discovering certain enemies before they get such information about them. So I like the name of the caster to not be spoiled in the log.

From another POV you can think of it that you (the battle brothers) don't hear the command because the commander is too far away. And the log just represents what you see/hear/smell/feel.

Reply:
Yes that's fine. But in your implementation you show both the user's name and the target's name even when only the user is visible. I suggest to have two conditions:
1. If only user is visible then it should say `user uses command`
2. If both user and target are visible then `user uses command on target`.